ui/src/markup.tsx

```diff
@@ -59,22 +62,26 @@ interface State {
    * :value "<div/>"
    **/
   content: S
+  /** 
+   * True if title and padding should be removed. Defaults to False.
+   **/
+  compact?: B
 }
 
 export const
   XMarkup = ({ model: { content, name } }: { model: Markup }) => (
     <div data-test={name} dangerouslySetInnerHTML={{ __html: content }} />
   ),
-  MarkupCard = ({ name, title, content }: { name: S, title: S, content: S }) => (
-    <div data-test={name} className={css.card}>
-      {title && <div className='wave-s12 wave-w6'>{title}</div>}
+  MarkupCard = ({ name, title, content, compact }: { name: S, title: S, content: S, compact?: B }) => (
+    <div data-test={name} className={clas(css.card, compact ? '' : css.cardPadding)}>
+      {!compact && title && <div className='wave-s12 wave-w6'>{title}</div>}
```

Comment:
Let's not make `title` dependent on `compact`, might be confusing. The title should be always rendered if specified, despite looking odd.